org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -151,6 +152,7 @@ private float MaxComponent(Vector3 v)
         /// <summary>
         /// Occludes the handle so it is not initially visible when it should start disabled.
         /// </summary>
+        [Obsolete("Force hiding handles is no longer supported. Use IsOccluded instead.")]
```

Comment:
ack .... please fix my typo :)

"Force hiding is no longer supported. Use IsOccluded instead."